patch/MultiDelay/MultiDelay.cpp

```diff
@@ -0,0 +1,138 @@
+#include "daisysp.h"
+#include "daisy_patch.h"
+#include <string>
+
+#define MAX_DELAY static_cast<size_t>(48000 * 1.f)
+
+using namespace daisy;
+using namespace daisysp;
+
+DaisyPatch patch;
+
+DelayLine <float, MAX_DELAY> DSY_SDRAM_BSS delMems[3]; 
+
+struct delay
+{
+    DelayLine <float, MAX_DELAY>  *del;
+    float currentDelay;
+    float delayTarget;
+};
+
+delay delays[3];
+Parameter params[4];
+
+float feedback;
+int drywet;
+
+void ProcessControls();
+
+float UpdateDelay(delay &delay, float in)
+{
+    //set delay times
+    fonepole(delay.currentDelay, delay.delayTarget, .004f);
```

Comment:
Most of the left over distortion from the delay time changing seems to go away with a coefficient around `0.0002f` -- It's easy to forget to drastically decrease this when moving from the 1kHz loop to the 48kHz loop, and changes in the delay time can make higher feedback settings quite muddy.

At some point we can make a delayline class that has some buffering to swap delay times without the pitch shifting effect.